docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: "3.8"
```

Comment:
Shouldn't this file and the addition to the contribution be in PR #164?

Reply:
Yeah it is, it pop up here because I changed the base and didn't rebase it here :p